docs/test/build.md

```diff
@@ -24,6 +24,9 @@ make unit-test
 make bdd-test
 ```
 
+## Running BDD tests in detached mode
```

Comment:
Feels like these lines should be added in [bdd_test.md](https://github.com/hyperledger/aries-framework-go/blob/master/docs/test/bdd_tests.md#run-bdd-tests) file.

Reply:
ok..

Reply:
done